examples/serve-inference/aws_neuron_core_inference_serve.py

```diff
@@ -0,0 +1,59 @@
+from fastapi import FastAPI  # noqa
+from ray import serve  # noqa
+
+import torch # noqa
+import os
+from transformers import AutoTokenizer # noqa
+
+app = FastAPI()
+
+hf_model = "openlm-research/open_llama_3b"
```

Comment:
Can we change this to llama2 instead of open-llama?

Reply:
I would prefer to have llama2 (commented). Default to open_llama as we don't want customers to wait for license approvals.